packages/blunderbuss/src/blunderbuss.ts

```diff
@@ -54,15 +54,15 @@ export = (app: Application) => {
       'pull_request.labeled',
     ],
     async context => {
-      const config = (await context.config(
-        CONFIGURATION_FILE_PATH,
-        {}
-      )) as Configuration;
-      const issue = context.issue() as Issue;
+      // const config = (await context.config(
+      //   CONFIGURATION_FILE_PATH,
```

Comment:
we should revert any of the `blunderbus` changes that were on this branch.

Reply:
`git checkout origin/master [filename]`

:point_up: you can just do something like this and checkout the specific files from a specific branch.